osm2streets/src/transform/intersection_geometry.rs

```diff
@@ -1,36 +1,41 @@
 use abstutil::Timer;
 use geom::{Circle, Distance};
 
-use crate::{IntersectionControl, StreetNetwork};
+use crate::{Road, StreetNetwork};
 
 pub fn generate(streets: &mut StreetNetwork, timer: &mut Timer) {
-    // TODO intersection_polygon assumes untrimmed lines as input, so reset here. Once we always
```

Comment:
This is now redundant, because `to_input_road` doesn't use it, and we set `road.center_line` at the end of this file